app/src/lib/directory-exists.ts

```diff
@@ -0,0 +1,10 @@
+import { stat } from 'fs/promises'
+
+/**
+ * Helper method to stat a path and check both that it exists and that it's
+ * a directory.
+ */
+export const directoryExists = (path: string) =>
+  stat(path)
+    .catch(null)
```

Comment:
I think this should be a function, right? 🤔 
```suggestion
    .catch(() => null)
```

Reply:
I see `catch` accepts `null` as a parameter, and the compiler didn't complain with your `s?` below, even though `s` is never `null` or `undefined` in your implementation 🤯